### ISSUE TITLE
zoom buttons appear on opening sidebar

### ISSUE THREAD

**Issue by Djfe** [mozilla/MozStumbler#976]
the gingerbread style zoom buttons of osmdroid appear every time I want to access the sidebar by using the swype gesture

is there a way to prevent this or is it an issue of OSMdroid?

**Comment by Djfe** [comment]
there was the idea to use the web view, maybe this way this could be solved and an alternative zoom bar could be introduced

**Comment by cascheberg** [comment]
This bug was not overly annoying, but only until #1197 landed...

**Comment by Djfe** [comment]
now that @crankycoder has changed so much OSMdroid code already,
it shouldn't be an issue to change the zoom buttons so that they fit more into the Holo/dark style of the app, maybe @ryanfeeley has an idea for the design
scale plus zoom buttons would look nice, some inspiration from the maps app:
![maps](http://cloud2.baohe.org/allimg/androidmobiletradarcom-1347779884/google-maps-updated-scale-bar-and-zoom-buttons-moved-from-labs-into-official-settings_1.png)

@crankycoder would need to make it so that osmdroid doesn't react to swypes, if an overlying activity gets activated
or the zoom buttons get moved to the right side of the screen instead

**Comment by santiago-hollmann** [comment]
@cascheberg could yout tell me when do they appear? For example, everytime you touch anything inside the drawer (such as metrics, bottom butoons) or the empty space between seetings and upload

**Comment by cascheberg** [comment]
@santiago-hollmann The zoom buttons only appear when opening the sidebar, as said in the first comment.
I assume my comment was a bit misleading. I did not mean to say that your PR made it happen more often. It's just that the zoom buttons perfectly hide the settings button now, so I always have to wait until the zoom buttons disappear before I can access the settings.
![screenshot_2014-11-05-19-12-22](https://cloud.githubusercontent.com/assets/3845150/4923053/5f51bc8a-6517-11e4-8acc-aa665715379a.png)

**Comment by santiago-hollmann** [comment]
@cascheberg no worries man, I asked because I saw it doing the actions I mentioned above and I fixed it so I was wondering if the issue still remains.

Just to get a little bit mre of context, could you tell me your device and OS version so I can replicate it. Thanks!

**Comment by cascheberg** [comment]
@santiago-hollmann Nexus 4 with latest stock Android 4.4.4

**Comment by santiago-hollmann** [comment]
@cascheberg thanks for the data.

Sorry for the delay, the last two nights I had no electricity :-1: 

I just switched to the latest commit (e445a39f5d611fffd0a9fa9b4273d8125efd6c3b) before #1197 and I found that on Android 4.4.4 (Nexus 5) the zoom buttons appear with the same frequency as they do today, which is every time you open the drawer with the gesture.

So, this could be an issue of the current map lib that we are using.
